pages/GPS-Activation-Device.md

```diff
@@ -0,0 +1,9 @@
+
+## Obtaining
+GPS Activation Device can be crafted in groups of eight in an [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table).
+
+## Usage
+GPS Activation Device is a component of a [Teleporter](https://github.com/Slimefun/Slimefun4/wiki/Teleporter).
+
+## Variations
+GPS Activation Device has two variations, personal and shared. Personal Activation Device can only be used by the one who placed it, while the Shared Activation Device can be used by anyone.
```

Comment:
by anyone who is permitted to activate pressure plates in that area.

Important detail because protection plugins may block someone.